microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -609,6 +610,24 @@ To enhance the Kids Mode experience, Microsoft Edge adds a safe keyword to the h
 Kids Mode does not collect data for personalization of the news feed or other Microsoft services.  You may not change the privacy settings for Kids Mode.  Other settings like Windows Defender SmartScreen and diagnostic data are configured according to the profile in which Kids Mode was launched.  For more information about diagnostic data about how you use the browser and Windows Defender SmartScreen, see the [Diagnostic Data](#diagnostic-data) and [SmartScreen](#smartscreen) sections.
 
 
+<!-- ====================================================================== -->
+## Magnify Image
+
+To use the Magnify Image feature, right-click an image and then select **Magnify image**, or press the **Ctrl** key twice while hovering over an image.  The Magnify Image feature retrieves the largest available version of the image from the website and displays it in a popup window.
+
+
+<!-- ------------------------------ -->
+#### Turning off the keyboard shortcut
+
+To turn off the **Ctrl-Ctrl** keyboard shortcut, go to `edge://settings/content/MagnifyImage`.  Or, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Cookies and site permissions** > **Magnify image**, and then use the **Enable Magnify keyboard shortcut** toggle.
+
+
+<!-- ------------------------------ -->
+#### AI-based image enhancement
+
+When you click the **AI enhance** button, AI-based image enhancement is done on your device by using a model that's downloaded from Microsoft servers.  Neither the image nor any personally identifiable information is sent to Microsoft.
```

Comment:
Where is the AI enhance button? How do users access it?

Reply:
![image](https://github.com/user-attachments/assets/8d3d612f-44c9-4a6f-8f71-eb49280569ce)
```
When you click the **AI enhance** button at the bottom of the popup window, AI-based image enhancement is done on your device by using a model that's downloaded from Microsoft servers.
```
@apryzgoda